client/app/bundles/course/forum/components/buttons/GenerateReplyButton.tsx

```diff
@@ -77,28 +81,36 @@ const GenerateReplyButton: FC<Props> = ({ post, forumId, topicId }: Props) => {
   }, [isLoading]);
 
   return (
-    <Tooltip title={t(translations.GenerateReplyTooltip)}>
-      <Chip
-        classes={{
-          root: 'pr-2',
-        }}
-        color="primary"
-        deleteIcon={
-          isLoading ? <LoadingIndicator bare size={15} /> : <AutoAwesome />
-        }
-        disabled={isLoading}
-        label={
-          isLoading
-            ? t(translations.GeneratingReply)
-            : t(translations.GenerateReply)
-        }
-        onClick={handleGenerateNewAnswer}
-        onDelete={(e) => {
-          e.stopPropagation();
-          handleGenerateNewAnswer();
-        }}
-        size="small"
-      />
+    <Tooltip
+      title={
+        post.isAiGenerated
+          ? t(translations.GenerateReplyDisabledTooltip)
+          : t(translations.GenerateReplyTooltip)
+      }
+    >
+      <span>
```

Comment:
Why's a `<span>` introduced here?

Reply:
To make sure tool tip can appear even when chip is disabled.